Sources/COWMacros/COWMacros.swift

```diff
@@ -5,7 +5,8 @@ import SwiftSyntaxMacros
 @_implementationOnly import SwiftSyntaxBuilder
 @_implementationOnly import SwiftDiagnostics
 
-private let defaultStorageName: TokenSyntax = "_$storage"
+let defaultStorageTypeName: TokenSyntax = "_$COWStorage"
+let defaultStorageName: TokenSyntax = "_$storage"
```

Comment:
Explicit `private` access control is recommended. This convention is learned from the Swift Standard Library.